testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5415,6 +5415,31 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("array_distinct(null)");
   }
 
+  @Test void testArrayJoinFunc() {
+    final SqlOperatorFixture f0 = fixture();
+    f0.setFor(SqlLibraryOperators.ARRAY_JOIN);
+    f0.checkFails("^array_join(array['aa', 'b', 'c'], '-')^", "No match found for function"
+        + " signature ARRAY_JOIN\\(<CHAR\\(2\\) ARRAY>, <CHARACTER>\\)", false);
+
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SPARK);
+    f.checkScalar("array_join(array['aa', 'b', 'c'], '-')", "aa-b-c",
+        "VARCHAR NOT NULL");
+    f.checkScalar("array_join(array[null, 'aa', null, 'b', null], '-', 'empty')",
+        "empty-aa-empty-b-empty", "VARCHAR NOT NULL");
+    f.checkScalar("array_join(array[null, 'aa', null, 'b', null], '-')", "aa-b",
+        "VARCHAR NOT NULL");
+    f.checkScalar("array_join(array[null, x'aa', null, x'bb', null], '-')", "aa-bb",
```

Comment:
What do you mean x'aa' ? Looks like some skill I don't know.

Reply:
@NobiGo  
 Hexadecimal Literals
https://docs.oracle.com/cd/E17952_01/mysql-8.0-en/hexadecimal-literals.html

for example
```
spark-sql (default)> select x'61';
a
```

Reply:
I get it. Thanks.